climakitae/threshold_panel.py

```diff
@@ -29,112 +53,67 @@ class ThresholdDataParams(param.Parameterized):
     smoothing = param.ObjectSelector(default="None", objects=["None", "Running mean"], label = "Smoothing")
     num_timesteps = param.Integer(default=10, bounds=(0, None), label = "Number of timesteps")
 
-    units2 = param.ObjectSelector(objects=dict())
-    # variable2 = param.ObjectSelector(default="Air Temperature at 2m", objects=["Air Temperature at 2m"])
-    variable2 = param.ObjectSelector(default="Air Temperature at 2m", objects=dict())
-    cached_area2 = param.ObjectSelector(default="Sacramento County", objects=dict())
-    area_subset2 = param.ObjectSelector(default="CA counties", objects=["CA counties", "states"])
-
     def __init__(self, *args, **params):
         super().__init__(*args, **params)
         
         # Selectors defaults
         self.selections.append_historical = False
         self.selections.area_average = True
-        self.selections.resolution = "45 km"
+        self.selections.resolution = "45km"
```

Comment:
Seems to me that a space is appropriate grammar-wise, since this is used in display text?

Reply:
Personally I prefer it without the space. Perhaps we can ask the larger group and see what everyone thinks?

Reply:
Addressed in PR https://github.com/cal-adapt/climakitae/pull/114